### PR TITLE
App controllers

### PR DESCRIPTION
**Pull request by carreraprogrammer**
-  For each URL you should create:
    - [x] A route.
    - [x] An action in the correct controller.
    - [x] A view file.

-  View files should include only placeholders written in plain HTML, e.g:
    - [x] Here is a list of posts for a given user

-  [x] Make sure that you added code to handle **only** the 4 required URLs.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
+  resources :users, only: [:index, :show]
+  resources :posts, only: [:index, :show]
```
Comment (mckent05) [inline config/routes.rb]: - Good job done so far, but your routes have not been properly generated. Your post route should be nested under the user route. Please fix this 🙏 . It should look something like this 

```ruby
 resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @carreraprogrammer, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 
- Good job setting up routes and controllers for your app 👍 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 1 above)
Comment (carreraprogrammer) [reply config/routes.rb]: Thanks for the correction, I did the changes

**Review by KanzaTahreem — APPROVED** [review]
# Approved 🙌

Hi @carreraprogrammer 

Good job implementing the suggested changes 💪 **_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Controllers are created with actions!
-  ✅ Routes are correctly setup, well done!
-  ✅ Your README looks good, well done!
-  ✅ Linters are working properly, great job! 

## Optional suggestions ❕

- [OPTIONAL] Well done for setting up README 📌✨ But I noticed that LICENSE is missing from your REAMDE. Kindly make sure to include it, making it easy for users and contributors to access and understand the license terms 🙌

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**